revolut/__init__.py

```diff
@@ -233,11 +233,11 @@ def orders(
         orders = []
         reqdata = {}
         if from_date:
-            from_date = dateutil.parser.parse(from_date)
-            reqdata["from_created_date"] = from_date.strftime("%y-%m-%dT%H:%M:%S.%f%z")
+            from_date = utils._date(from_date).isoformat()
+            reqdata["from_created_date"] = from_date.strftime("%Y-%m-%dT%H:%M:%S.%f%z")
```

Comment:
Skoro API akceptuje `datetime` to w ogóle przydałby się odpowiednik `utils._date` dla tego typu, np. `utils._datetime`. Nie rozumiem natomiast całego flow. Wydaje mi się, że wystarczyłoby `reqdata["from_created_date"] = utils._datetime(since).isoformat()`